### PR TITLE
Update UserGuide.md

### PR DESCRIPTION
**Pull request by yellow-294**

### CODE REVIEW

**Review by vantemoon — COMMENTED** [review]
Other issues related to case sensitivity that can be added:
1. Positions are case-insensitive
2. Availabilities are case-insensitive
3. Roles are case-insensitive

Regarding issue #138, can add "grades should be in the format [integer]/[integer] in the grade command section.

**Review by cxyterence — APPROVED** [review]
LGTM, but add on to above comments

File: docs/UserGuide.md [hunk 1]
```diff
@@ -87,9 +91,22 @@ Format: `add n/NAME p/PHONE_NUMBER e/EMAIL pos/POSITION a/ADDRESS t/MODULE-GROUP
...
+    * end with a domain label at least 2 characters long
+    * have each domain label start and end with alphanumeric characters
+    * have each domain label consist of alphanumeric characters, separated only by hyphens, if any.
+* `Position`: `Student`, `TA`, or `Professor`
+* `Address`: Any value, but cannot be blank
+* `Tags`: {Alphanumeric Module}-{Alphanumeric Tutorial group}
 </div>
 
-Adds a new contact with the provided details. Required fields include name, phone number, email, and position. Tags representing the module and tutorial group a person is associated with are optional and must be in the format of module-tutorial group.
+Adds a new contact with the provided details. Required fields include name, phone number, email, position, and tag. Tags represent the module and tutorial group a person is associated with and must be in the format of module-tutorial group.
```
Comment (guowei42) [inline docs/UserGuide.md:109]: I think should add address is required field as well

**Review by guowei42 — CHANGES_REQUESTED** [review]
other than the one part commented below LGTM

**Comment by yellow-294** [comment]
Respective changes have been made, thanks everyone.

**Review by guowei42 — APPROVED** [review]

**Pull request merged**